docs/tutorials/documenting-api-endpoints.md

```diff
@@ -0,0 +1,121 @@
+# Documenting REST API endpoints
+
+This document briefly explains how to document Zulip's REST API endpoints
+on our `/api` page.
+
+Our API documentation files live under `templates/zerver/api/*`. To begin,
+we recommend copying over an existing doc file (`render-message.md` is a good
+example) and using it as a template. Make sure you link to your new Markdown
+file in `templates/zerver/help/rest-endpoints.md`, so that it can be displayed
+in the sidebar on the `/api` page.
+
+## Usage examples.
```

Comment:
I think it's better if we remove the period here, as we don't use it for titles along the docs.

Reply:
Agree on removing the period for better consistency

Reply:
This is a typo, so it wasn't intended lol! :)